ci/publish-book.sh

```diff
@@ -16,22 +16,41 @@ echo --- create book repo
   git commit -m "${CI_COMMIT:-local}"
 )
 
-eval "$(ci/channel-info.sh)"
-# Only publish the book from the edge and beta channels for now.
-case $CHANNEL in
-edge)
-  repo=git@github.com:solana-labs/book-edge.git
-  ;;
-beta)
-  repo=git@github.com:solana-labs/book.git
-  ;;
-*)
- echo "--- publish skipped"
- exit 0
- ;;
-esac
+if [[ "$MANUAL_PUBLISH" = true ]]; then
+  case $BOOK in
+  book)
+    repo=git@github.com:solana-labs/book.git
+    ;;
+  book-edge)
```

Comment:
Now that I'm looking at it, not sure the mechanism for manually pushing these books is needed.  It would get over-written as soon as someone makes a new commit to the branch...is there a case where this would be needed?

Reply:
Yep I think we can let the machines fully own -edge and -beta